### ISSUE TITLE
Settings Menu

### ISSUE THREAD

**Issue by ChristopherJMiller** [ChristopherJMiller/temple#14]
Settings should be able to manage audio channel volume and accessibility options.

**Comment by ChristopherJMiller** [comment]
Audio Channel settings added in d42047b95c906bfaea762788448d284385d50765
